src/calculateResult.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const validateInput = require('./validateInput');
+
+function calculateResult(inputNumber, randomNumber) {
+  const result = {
+    cows: 0,
+    bulls: 0,
+  };
+
+  const validationResult = validateInput(inputNumber);
+
+  if (validationResult) {
+    return validationResult;
+  }
+
+  for (let i = 0; i < inputNumber.length; i++) {
```

Comment:
Consider using forEach instead of a traditional for loop for iterating over the inputNumber array.